README.md

```diff
@@ -9,6 +9,7 @@ Django-based application to manage registration of people for [scientific summer
 - `source venv/bin/activate` - activate venv
 - `./migrate.sh` - apply DB migrations
 - `./manage.py createsuperuser` - script to create a superuser that can modify DB contents via admin panel
+- `./manage.py populate_db` - script to populate the database with data for development
```

Comment:
Rename to so it's evident this creates test data. `populate_with_test_data` or `create_fake_data` or `create_test_data` or something else.